spharpy/beamforming/beamforming.py

```diff
@@ -197,31 +236,49 @@ def maximum_front_back_ratio_weights(n_max, normalize=True):
 
     Examples
     --------
-    Apply weights maximizing the front-back ratio to a plane wave
-    decomposition. The direction of arrival is zero degrees azimuth.
+    Apply beamformers maximizing the front-back ratio to a plane wave from
+    zero degrees azimuth.
 
     .. plot::
 
         >>> import spharpy
         >>> import pyfar as pf
         >>> import matplotlib.pyplot as plt
         >>> import numpy as np
-        >>> N = 5
-        >>> steering = pf.Coordinates.from_spherical_colatitude(
-        ...     np.linspace(0, 2*np.pi, 500), np.ones(500)*np.pi/2,
-        ...     np.ones(500))
-        >>> Y_steering = spharpy.spherical.spherical_harmonic_basis_real(
-        ...     N, steering)
-        >>> a_nm = spharpy.spherical.spherical_harmonic_basis_real(
-        ...     N, pf.Coordinates(1, 0, 0))
-        >>> d_nm = spharpy.beamforming.maximum_front_back_ratio_weights(N)
-        >>> beamformer = np.squeeze(Y_steering @ np.diag(d_nm) @ a_nm.T)
+        >>>
+        >>> # use real valued spherical harmonics of order 7
+        >>> definition = spharpy.SphericalHarmonicDefinition(n_max=7)
+        >>>
+        >>> # define the sound field (plane wave from 0 degree azimuth)
+        >>> soundfield = spharpy.SphericalHarmonics.from_definition(
+        ...     definition, pf.Coordinates(1, 0, 0))
+        >>> a_nm = soundfield.basis
+        >>>
+        >>> # beamforming
+        >>> # a) generate 500 steering vectors
+        >>> steering = spharpy.SphericalHarmonics.from_definition(
+        ...     definition, pf.Coordinates.from_spherical_elevation(
+        ...         np.linspace(0, 2*np.pi, 500), 0, 1))
+        >>> Y_steering = steering.basis
+        >>>
+        >>> # b) design beamformer weights
+        >>> R = 10**(50/20)
+        >>> d_nm = spharpy.beamforming.maximum_front_back_ratio_weights(
+        ...     definition.n_max)
+        >>>
+        >>> # c) appling beamformer weights yields 500 beamformers
+        >>> beamformer = np.squeeze(Y_steering @ np.diag(d_nm))
+        >>>
+        >>> # d) apply beamformers to the soundfield
+        >>> soundfield_beamformed = beamformer @ a_nm.T
+        >>>
+        >>> # plot soundfield evaluated with beamformers
         >>> ax = plt.axes(projection='polar')
-        >>> ax.plot(steering.azimuth, 20*np.log10(np.abs(beamformer)))
-        >>> ax.set_rticks([-75, -50, -25, 0])
+        >>> ax.plot(steering.coordinates.azimuth,
+        ...         20*np.log10(np.abs(soundfield_beamformed)))
+        >>> ax.set_rticks([-50, -25, 0])
```

Comment:
With the new radial ticks I'd also suggest to change the limits. Otherwise the plot looks a bit weird.